python/GafferUI/NodeUI.py

```diff
@@ -122,11 +122,8 @@ def node( self ) :
 
 	## Should be implemented by derived classes to return
 	# a PlugValueWidget they are using to represent the
-	# specified plug. Since many UIs are built lazily on
-	# demand, this may return None unless lazy=False is
-	# passed to force creation of parts of the UI that
-	# otherwise are not yet visible to the user.
-	def plugValueWidget( self, plug, lazy=True ) :
+	# specified plug.
+	def plugValueWidget( self, plug ) :
 
 		return None
```

Comment:
What are the valid use cases to return `None` now? Might be good to document them or should it raise?

Reply:
There will be no widget because when UI metadata says there shouldn't be one, in which case we should return `None` and let the caller figure out what's best. I believe our most common use cases are things like "show the widget for this plug if there is one, if not don't worry".

Reply:
Documented in d16726a.